src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs

```diff
@@ -136,7 +136,7 @@ public override IReadOnlyList<Type> GetParameterTypes(string methodName)
         {
             if (!_methods.TryGetValue(methodName, out var descriptor))
             {
-                return Type.EmptyTypes;
+                throw new HubException("Method doesn't exist.");
```

Comment:
Include the name of the method that doesn't exist

Reply:
It's already in the message that gets generated. See the tests.

Reply:
Derp